index.js

```diff
@@ -68,8 +71,9 @@ app.post('/v1/autodock', (req, res) => {
         fs.mkdirSync(uploadDirectory);
     }
     else {
-        res.status(400);
         errorMsg = 'Hash collision.';
+        res.status(400)
+        return res.send(errorMsg)
```

Comment:
There's an issue with formidable where it will attempt to send a response whether the binary execution fails or not. So Setting the errorMsg is the mechanism I use to determine an error condition - so remove res.send.

Reply:
Sending a response after a response was already sent will cause node to fail

Reply:
That's why I'm returning here, to prevent execution from continuing on to where it might cause a failure.

Reply:
So here it should be fine, outside of the `form.on` call. It won't get repeated at all, and it'll actually end execution.